imagetest/test_suites/packagevalidation/package_test.go

```diff
@@ -93,14 +93,11 @@ func TestGuestPackages(t *testing.T) {
 		requiredPkgs = append(requiredPkgs, "epel-release")
 	}
 	if strings.Contains(image, "debian") {
-		requiredPkgs = removeFromArray(requiredPkgs, "google-cloud-sdk") // debian has google-cloud-cli
-		requiredPkgs = append(requiredPkgs, "google-cloud-cli")
 		requiredPkgs = append(requiredPkgs, "haveged", "net-tools")
 		requiredPkgs = append(requiredPkgs, "google-cloud-packages-archive-keyring", "isc-dhcp-client")
 		excludePkgs = append(excludePkgs, "cloud-initramfs-growroot")
 	}
 	if strings.Contains(image, "ubuntu") {
-		requiredPkgs = removeFromArray(requiredPkgs, "google-cloud-sdk")
```

Comment:
On Ubuntu, there isn't a package installed. There is a snap for google-cloud-cli (or sdk) instead. Ideally we would look to see if that snap is preseeded however that can be later.

Reply:
Removed the check for ubuntu and added a TODO here.